Lib/test/test_dict_version.py

```diff
@@ -80,14 +80,14 @@ def test_setitem_same_value(self):
 
         # setting a key to the same value with dict.__setitem__
         # must change the version
-        self.check_version_changed(d, d.__setitem__, 'key', value)
```

Comment:
@zware As far as git log, these tests are committed by you.
Do you remember why setting same value should increment the version?

Reply:
I can think of reasons to prefer it either way ... does the existence of this test mean there is a backwards compatibility note needed?

Reply:
No.  This test is implemented with `_testcapi`, which is backdoor for testing internals.
No one use it from public stable API.